### PR TITLE
revising reporting dir

### PR DESCRIPTION
**Pull request by david-piiano**

### CODE REVIEW

File: cli/version.json [hunk 1]
```diff
@@ -1 +1 @@
-{ "engine" : "0.4.484", "viewer" : "0.2.132-65" }
+{ "engine" : "0.4.530", "viewer" : "0.2.139-89" }
```
Comment (nirhaas) [inline cli/version.json:1]: Counting on you that it's corerct

File: cli/flows-cli.sh [hunk 2]
```diff
@@ -260,25 +261,32 @@ if [ ${RUN_VIEWER} = "skip" ] ; then
...
+if [ ! -e ${SCAN_OUTPUT_DIR}/report.json ] ; then
+  echo "ERROR: expecting report.json file to be at ${SCAN_OUTPUT_DIR}/report.json"
   exit 1
 fi
-cp -f ${OUTPUT_DIR}/report.json ${OUTPUT_DIR}/api/offline-report.json
+
+# Create a report directory with a placeholder for 'previous'
+# Move all existing reports into a unique 'previous' directory
+REPORT_DIR=${BASEDIR}/.report
+OLDER_FOLDER=$(date "+%y%m%d_%H%M%S")
```
Comment (nirhaas) [inline cli/flows-cli.sh:273]:
```suggestion
OLDER_FOLDER=$(date "+%s")
```

File: cli/flows-cli.sh [hunk 3]
```diff
@@ -260,25 +261,32 @@ if [ ${RUN_VIEWER} = "skip" ] ; then
...
+mkdir -p ${REPORT_DIR}/${OLDER_FOLDER}
+mkdir -p ${REPORT_DIR}/api
```
Comment (nirhaas) [inline cli/flows-cli.sh:275]:
```suggestion
mkdir -p ${REPORT_DIR}/${OLDER_FOLDER} ${REPORT_DIR}/api
```

File: cli/flows-cli.sh [hunk 4]
```diff
@@ -260,25 +261,32 @@ if [ ${RUN_VIEWER} = "skip" ] ; then
...
+mv ${REPORT_DIR}/api/*.json ${REPORT_DIR}/${OLDER_FOLDER} > /dev/null >&2 || true
```
Comment (nirhaas) [inline cli/flows-cli.sh:276]: Why only json files? move everything

**Review by nirhaas — APPROVED** [review]
Hope you tested it

File: cli/flows-cli.sh (hunk 2 above)
Comment (david-piiano) [reply cli/flows-cli.sh:273]: epoch time is unreadable, so let's keep what I suggested

File: cli/flows-cli.sh (hunk 3 above)
Comment (david-piiano) [reply cli/flows-cli.sh:275]: agreed, changing

File: cli/flows-cli.sh (hunk 4 above)
Comment (david-piiano) [reply cli/flows-cli.sh:276]: Historical, irrelevant reason. Changing to move *

File: cli/version.json (hunk 1 above)
Comment (david-piiano) [reply cli/version.json:1]: confirmed with guy, so yes

**Pull request merged**